app/models/project/export/info_request.rb

```diff
@@ -0,0 +1,60 @@
+##
+# Export key classification and extracted data for an info request within a
+# given project.
+#
+class Project::Export::InfoRequest
+  extend Forwardable
+
+  attr_reader :project, :info_request
+
+  def initialize(project, info_request)
+    @project = project
+    @info_request = info_request
+  end
+
+  def data
+    {
+      request_url: info_request.url_title,
```

Comment:
Should be the full URL rather than just the slug

Reply:
Doh, of course changed!